scylla-cql/Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories = ["database"]
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-scylla-macros = { version = "0.7.0", path = "../scylla-macros" }
+scylla-macros = { version = "=0.7.0", path = "../scylla-macros" }
 byteorder = "1.3.4"
```

Comment:
:wrench: Commit: `scylla-cql: Depend on exact version of scylla-macros`: future blame's readers will appreciate putting justification of this change in the commit message.